Community/ideAlarm/automation/lib/python/community/idealarm/__init__.py

```diff
@@ -362,7 +364,7 @@ def __init__(self):
         for alarmZone in self.alarmZones:
             alarmZone.getNagSensors()
 
-        self.log.info("ideAlarm object initialized with {} zones at {}".format(len(self.alarmZones), self.timeCreated))
+        self.log.info("ideAlarm object initialized with {} zones at {}".format(len(self.alarmZones), format_date(self.timeCreated, customDateTimeFormats['dateTime'])))
```

Comment:
Update the configuration.py.example for ideAlarm like you did for weatherStationUploader.

Reply:
I actually put in into the main configuration file  Core/automation/lib/python/configuration.py.example since I thought it will be quite useful for several Community scripts.

Would you prefer to put it in each community scripts configuration file instead? 

I will continue tomorrow morning.

Cheers!

Reply:
> Would you prefer to put it in each community scripts configuration file instead?

Yes, please. Each community submission should include everything it will need to run. Things should only go into /Core/automation/lib/python.configuration.py if something in core needs it.

Reply:
Good morning. We'd have to decide where to put configuration entries that is needed for a community script.

I guess that when the configuration entry only is needed in a single community script we put it in that script's configuration.py.example file.

I suggest that when a configuration entry is needed in several community scripts we add it in the main configuration.py.example file and not in any of the configuration.py.example files belonging to the using scripts. Putting a copy of a configuration entry in several places will probably lead to that some of them get outdated sooner or later.

What do you suggest?

Reply:
It's getting late here, but good morning to you! 

If a community module/script needs something in configuration.py, then it should have a configuration.py.example with just what it needs. Even if the same thing is needed in another community module/script. If we put all of them into Core configuration.py, it will get filled with stuff that people don't use. This also keeps everything neat and tidy in a single directory. 

I had put openhabHost and openhabPort in there as examples to get people started but they probably shouldn't be there. 

> Putting a copy of a configuration entry in several places will probably lead to that some of them get outdated sooner or later.

I don't understand how that would be possible.

Reply:
I agree. Fixed as proposed.